modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -19,8 +19,21 @@ IMPORTANT: OpenShift oc executable must be located in a system location that is
 +
 IMPORTANT: Ensure that you can run link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#getting-started[oc cluster up] with no errors before moving on to the installation of AeroGear.
 +
-
+* Docker (version 1.22 or above)
```

Comment:
fatal: [localhost]: FAILED! => {"changed": false, "cmd": "oc cluster up --service-catalog=true --host-data-dir=/home/jstaffor/mext/mobile-core/ui/openshift-data --routing-suffix=192.168.3.178.nip.io --public-hostname=192.168.3.178 --version=v3.9.0 --image=openshift/origin", "delta": "0:01:31.000149", "end": "2018-07-19 13:58:12.218949", "msg": "non-zero return code", "rc": 1, "start": "2018-07-19 13:56:41.218800", "stderr": "error: FAIL\n   Error: error pulling Docker image openshift/origin:v3.9.0\n   Caused By:\n     Error: Error: No such image: openshift/origin:v3.9.0", "stderr_lines": ["error: FAIL", "   Error: error pulling Docker image openshift/origin:v3.9.0", "   Caused By:", "     Error: Error: No such image: openshift/origin:v3.9.0"], "stdout": "WARNING: Docker version is 1.18, it needs to be >= 1.22\nPulling image openshift/origin:v3.9.0\nPulled 3/6 layers, 50% complete\nPulled 3/6 layers, 63% complete\nPulled 3/6 layers, 76% complete\nPulled 4/6 layers, 83% complete\nPulled 5/6 layers, 90% complete", "stdout_lines": ["WARNING: **Docker version is 1.18, it needs to be >= 1.22**", "Pulling image openshift/origin:v3.9.0", "Pulled 3/6 layers, 50% complete", "Pulled 3/6 layers, 63% complete", "Pulled 3/6 layers, 76% complete", "Pulled 4/6 layers, 83% complete", "Pulled 5/6 layers, 90% complete"]}

Reply:
Raised this with Openshift origin: https://github.com/openshift/origin/pull/20367

Reply:
I'm removing this as according to the OpenShift PR[1], there is no "there is no such thing as docker 1.22 (1.18 is the current latest version)." 

In addition, the error listed above "is coming from docker code and referencing the docker api version, not the docker client **version."[1].

[1] https://github.com/openshift/origin/pull/20367